sleap_io/model/instance.py

```diff
@@ -156,6 +142,7 @@ def _convert_points(self, attr, points):
     from_predicted: Optional[PredictedInstance] = None
 
     def __attrs_post_init__(self):
+        """Maintain point mappings between node and points after initialization."""
         super().__setattr__("points", self._convert_points(None, self.points))
```

Comment:
Could possibly use [converters](https://www.attrs.org/en/stable/init.html#converters), but may be problematic since we use an instance method `self._convert_points` to do the conversion
>Attributes can have a converter function specified, which will be called with the attribute’s passed-in value to get a new value to use.